docs/eventing/samples/apache-camel-source/README.md

```diff
@@ -11,64 +10,71 @@ for generating events.
    Documentation includes specific instructions for common Kubernetes
    environments, including development clusters.
 
-1. Install the Camel Source from the `camel.yaml` in the
-   [Eventing Sources release page](https://github.com/knative/eventing-contrib/releases):
+1. Install the Camel Source from the [yaml files in the config dir](../config/)
+   from source:
 
    ```shell
-   kubectl apply --filename camel.yaml
+   ko apply -f ../config/
    ```
+  
+  Or install a released version.   
+   
 
-## Create a Channel and a Subscriber
+### Create a Channel and a Subscriber
 
-In order to check if a `CamelSource` is fully working, we will create:
+To check that a `CamelSource` is fully working, we will create:
 
 - a simple Knative event display service that prints incoming events to its log
-- a in-memory channel named `camel-test` that will buffer events created by the
-  event source
-- a subscription to direct events from the test channel to the event display
-  service
+- an in-memory channel named `camel-test` that will buffer events created by the event source
+- a subscription to direct events from the test channel to the event display service
 
-Deploy the [`display_resources.yaml`](./display_resources.yaml):
+Deploy `display_resources.yaml`, building it from source:
 
 ```shell
-kubectl apply --filename display_resources.yaml
+ko apply -f display_resources.yaml
 ```
 
-## Run a CamelSource using the Timer component
+### Run a CamelSource using the Timer component
 
-The simplest example of CamelSource, that does not require additional
-configuration, is the "timer" source.
+The samples directory contains some sample sources that can be used to generate
+events.
+
+The simplest one, that does not require additional configuration is the "timer"
+source.
 
 If you want, you can customize the source behavior using options available in
 the Apache Camel documentation for the
-[timer component](https://github.com/apache/camel/blob/master/components/camel-timer/src/main/docs/timer-component.adoc).
+[timer component](https://github.com/apache/camel/blob/master/camel-core/src/main/docs/timer-component.adoc).
 All Camel components are documented in the
 [Apache Camel github repository](https://github.com/apache/camel/tree/master/components).
 
-Install the [`source_timer.yaml`](source_timer.yaml) resource:
+Install the [timer CamelSource](source_timer.yaml) from source:
 
 ```shell
-kubectl apply --filename source_timer.yaml
+ko apply -f source_timer.yaml
 ```
 
-We will verify that the published events were sent into the Knative eventing
-system by looking at what is downstream of the `CamelSource`.
+We will verify that the published message was sent to the Knative eventing
+system by looking at the downstream of the `CamelSource`.
 
-```shell
-kubectl logs --selector serving.knative.dev/service=camel-event-display -c user-container
-```
+1. Use [`kail`](https://github.com/boz/kail) to tail the logs of the subscriber.
```

Comment:
`kail` should also be added as a prerequisite.

Reply:
kubect logs instead of jail should be enough